### ISSUE TITLE
Responsive images on featured collection section.

### ISSUE THREAD

**Issue by kivlov** [Shopify/starter-theme#146]
I get an error from the lazysizes package because  it's failing to to pick up a width attribute from `data-src` attribute. 

`{%- assign img_url = image | img_url: '1x1' | replace: '_1x1.', '_{width}x.' -%}`

I cannot figure out were `_{width}x.` is being defined.

![image](https://user-images.githubusercontent.com/6742768/51980205-a7769300-248f-11e9-9a1e-a27a9182ab80.png)
![image](https://user-images.githubusercontent.com/6742768/51980242-bf4e1700-248f-11e9-9f92-f697cca4be02.png)

**Comment by rhandrade** [comment]
I found the problem and is not related directly with lazyload js. The problem is in `responsive-image.liquid` file. 

Look for `{%- assign image_widths = image_widths | strip_newlines -%}` line. `strip_newlines` filter should remove newlines from variable `image_widths`, but don't work. I replaced `strip_newlines` for `strip` that remove tabs, blank space and newlines. With that the lazyload script is able to generate and load images.

I think that should be fixed in next commit.
